gateway/1.5/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -2,4 +2,36 @@
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
-:imagesdir: ../assets/images
\ No newline at end of file
+:imagesdir: ../assets/images
+
+If you have completed all the steps in xref:policies-pdk-create-project.adoc[], you can compile the custom policy project.
+
+You must compile the custom policy before you can deploy it to a Flex Gateway.
+
+After you compile, test your policy with the following tutorials:
```

Comment:
```suggestion
After you compile your project, test your policy with the following tutorials:
```

Reply:
Not sure if it's project or policy, but you'll definitely want to specify what they're compiling.